### ISSUE TITLE
post training quantization using openvino pot or nncf

### ISSUE THREAD

**Issue by snehashis1997** [meituan/YOLOv6#884]
### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar feature requests.

### Description

post training quantization using openvino pot or nncf

### Use case

cpu inference

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by snehashis1997** [comment]
yes, can you pls list down the proper steps to do this PTQ using openvino ?

> Hi, what's your question, do you mean you want us to support PTQ in openvino?

**Comment by knoppmyth** [comment]
@snehashis1997 There are [docs](https://github.com/meituan/YOLOv6/blob/main/docs/Tutorial%20of%20Quantization.md) on this...
